packages/snaps-utils/src/snaps.ts

```diff
@@ -85,6 +89,10 @@ export type PersistedSnap = Snap;
  * A Snap as it exists in {@link SnapController} state.
  */
 export type Snap = TruncatedSnap & {
+  /**
+   * The initial connections of the Snap, optional, requested on installation.
+   */
+  initialConnections?: InitialConnections;
```

Comment:
This type is not correct unless you add `initialConnections` to `#set()` in the `SnapController`.

Reply:
Added it to `#set()` now.